server/graphql/v2/query/collection/OrdersCollectionQuery.ts

```diff
@@ -66,6 +68,11 @@ const getCollectivesJoinCondition = (
         ...(limitToHostedAccountsIds.length ? { [`$${association}.id$`]: { [Op.in]: limitToHostedAccountsIds } } : {}),
       },
     ];
+
+    if (excludeHostAccount) {
+      conditions.push({ [`$${association}.id$`]: { [Op.not]: account.id } });
+      conditions.push({ [`$${association}.ParentCollectiveId$`]: { [Op.not]: account.id } });
```

Comment:
All these conditions end up in an OR, so it looks like we're saying:
- Return all accounts
- _If one of the previous conditions match_
- **Or** if it's not the parent account
- **Or** if it's not a child of the parent account

As a result, it looks like we're going to return all accounts on the platform?